.github/workflows/test.yml

```diff
@@ -66,7 +66,7 @@ on:
       - feature
       - staging
       - prod
-      - release-2.0
+      - "release-*"
```

Comment:
No double quotes please.

Reply:
Ah I did this purposefully as this file still has double quotes across it everywhere int he prod version, but will update :D